db/migrate/20160713130940_remove_type_template_and_vms_filters_from_miq_search.rb

```diff
@@ -0,0 +1,33 @@
+class RemoveTypeTemplateAndVmsFiltersFromMiqSearch < ActiveRecord::Migration[5.0]
+  class MiqSearch < ActiveRecord::Base
+    serialize :filter
+  end
+
+  TEMPLATE_FITLER_EXPR = MiqExpression.new("=" => {"field" => "VmInfra-template", "value" => "true"})
```

Comment:
This is causing problems on master. You cannot use `MiqExpression` like this in migrations as it loads models from the application, which `good-migrations` prohibits. Is the `:filter` key even necessary below?